docs/pipeline-components-and-applications/enrichment-components/stream-enrich/index.md

```diff
@@ -20,34 +25,35 @@ A Terraform module is available which deploys Stream Enrich on a AWS EC2 without
 
 Stream Enrich is on docker hub with several different flavours. Pull the image that matches the source/sink you are using:
 
-```bash
-docker pull snowplow/stream-enrich-kinesis:3.2.3
-docker pull snowplow/stream-enrich-kafka:3.2.3
-docker pull snowplow/stream-enrich-nsq:3.2.3
-```
+<CodeBlock language="bash">{
+`docker pull snowplow/stream-enrich-kinesis:${versions.enrich}
```

Comment:
Are versions of Stream Enrich matching the new Enrich? Or do we need separate versioning?

Reply:
Yes all enrich assets use the same version.